codalab/worker/worker_run_state.py

```diff
@@ -276,9 +277,7 @@ def mount_dependency(dependency, shared_file_system):
         docker_image = run_state.resources.docker_image
         image_state = self.docker_image_manager.get(docker_image)
         if image_state.stage == DependencyStage.DOWNLOADING:
-            status_messages.append(
-                'Pulling docker image: ' + (image_state.message or docker_image or "")
-            )
+            status_messages.append(image_state.message)
```

Comment:
I think we should keep the "Pulling docker image [image_spec]" here (and remove it from docker_image_manage.py), just for consistency with the rest of the code.

Reply:
I see - but keep the percent stuff in the message in `docker_image_manager.py`